cfgov/jinja2/v1/_includes/organisms/_vars-mega-menu.html

```diff
@@ -200,9 +200,25 @@
     'footer': {
         'type': 'footer',
         'value': {
-            'content': 
-                '<p>Browse answers to hundreds of financial questions. <a href="/ask-cfpb/">Ask CFPB</a></p>
-                <p>Have an issue with a financial product? <a href="/complaint/">Submit a complaint</a></p>',
+            'content':
+                '<p>
+                    <span aria-hidden="true">
+                        Browse answers to hundreds of financial questions.
+                    </span>
+                    <a aria-label="Ask CFPB: browse answers to hundreds of financial questions."
```

Comment:
Honest wondering: is there a reason why the phrases are switched so "Ask CFPB" comes first before the contextual sentence?

Reply:
If the answer is not really, can we switch them around to match the visible text? That way, VoiceOver will lead with the phrase that's recognizable to people rather than the jargony product name.

Reply:
We can, but it sounds weird when read the other way around because it's not clear the jargony product name is not part of a longer missing sentence. The other link leads with a question `Have an issue with a financial product?`, while this one just reads `Browse answers to hundreds of financial questions.`. If I was speaking this I would say something like `Ask CFPB to browse answers to hundreds of financial questions.`

Reply:
Or `Browse answers to hundreds of financial questions with Ask CFPB.`

Reply:
I like that last option!

Reply:
Fixed to be `Browse answers to hundreds of financial questions with Ask CFPB.`